advanced_functionality/scikit_learn_bring_your_own_LightGBM_model/scikit_learn_bring_your_own_LightGBM_model.ipynb

```diff
@@ -0,0 +1,1121 @@
+{
```

Comment:
<p>Thanks for adding the deletion. It's standard to define a <code># Cleanup</code> section for this.</p><p> </p><p>Just before the Cleanup, optionally add a <code># Conclusion</code>  section summarizing what the customer accomplished and any recommended next steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3084/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Done.